test/connection_string/test/valid-host_identifiers.json

```diff
@@ -132,18 +132,18 @@
     },
     {
       "description": "UTF-8 hosts",
-      "uri": "mongodb://b\u00fccher.example.com,uml\u00e4ut.example.com/",
+      "uri": "mongodb://bücher.example.com,umläut.example.com/",
```

Comment:
To fix this error:
```
 [2021/10/14 04:12:08.559] ERROR [0.001s]: test_uri_spec (unittest.loader._FailedTest)
 [2021/10/14 04:12:08.559] ----------------------------------------------------------------------
 [2021/10/14 04:12:08.559] ImportError: Failed to import test module: test_uri_spec
 [2021/10/14 04:12:08.559] Traceback (most recent call last):
 [2021/10/14 04:12:08.559]   File "/opt/python/3.6/lib/python3.6/unittest/loader.py", line 428, in _find_test_path
 [2021/10/14 04:12:08.559]     module = self._get_module_from_name(name)
 [2021/10/14 04:12:08.559]   File "/opt/python/3.6/lib/python3.6/unittest/loader.py", line 369, in _get_module_from_name
 [2021/10/14 04:12:08.559]     __import__(name)
 [2021/10/14 04:12:08.559]   File "/data/mci/e81b7f83cb47c3e1dded2418195e2513/src/test/test_uri_spec.py", line 199, in <module>
 [2021/10/14 04:12:08.559]     create_tests(test_path)
 [2021/10/14 04:12:08.559]   File "/data/mci/e81b7f83cb47c3e1dded2418195e2513/src/test/test_uri_spec.py", line 181, in create_tests
 [2021/10/14 04:12:08.559]     scenario_def = json.load(scenario_stream)
 [2021/10/14 04:12:08.559]   File "/opt/python/3.6/lib/python3.6/json/__init__.py", line 296, in load
 [2021/10/14 04:12:08.559]     return loads(fp.read(),
 [2021/10/14 04:12:08.559]   File "/opt/python/3.6/lib/python3.6/encodings/ascii.py", line 26, in decode
 [2021/10/14 04:12:08.559]     return codecs.ascii_decode(input, self.errors)[0]
 [2021/10/14 04:12:08.559] UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 2902: ordinal not in range(128)
```

You need to change the JSON file `open` call to use `encoding='utf-8'`:
```
            with open(os.path.join(dirpath, filename), encoding='utf-8') as scenario_stream:
```

Reply:
Done.